enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+---
+title: audit-command-operation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-22
+last-updated: 2021-02-22
+status: implementable
+see-also:
+  - [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md)
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command to be added to SDK or OPM or in a new tool?
```

Comment:
IHMO we should at least initially add it on SDK. It does not mean that it cannot be indeed maintained by other teams. However, the reasons for that is because it would reduce a lot of the complexities and the effort required to get it done as its complexities. See:  https://github.com/operator-framework/enhancements/pull/66/files#r586452207 and https://github.com/operator-framework/enhancements/pull/66/files#r586458930

Reply:
@camilamacedo86 `audit` command does not make sense, at least to me, in the SDK. It seems very specific to a production catalog which SDK should not be dealing with. I agree with @estroz comment above.

Reply:
It was clarified in the open question. 

Also, it is important to highlight that the audit command also requires to perform check against specific Openshift criteria optionally which means that if it not be shipped in SDK or not use SDK binary as you @estroz highlight then the @joelanford suggestion to create in the interface where the specific Opeshift validator can only be used by SDK to address https://github.com/openshift/enhancements/pull/661 might not be so long the best approach. 

Then, we might need to have the OCP validator in the operator-framework api or its downstream component in order to use it and not have any relation with SDK at all if it is the case.

Reply:
Marking as solved because it is addressed already. However, please feel free to contact me if it still not clear at all.